recipe/build.sh

```diff
@@ -64,6 +68,8 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DBUILD_UTILITIES=ON \
       -DENABLE_DOXYGEN=OFF \
       -DENABLE_LOGGING=ON \
+      -DCMAKE_C_FLAGS_RELEASE=${CFLAGS} \
+      -DCMAKE_C_FLAGS_DEBUG=${CFLAGS} \
```

Comment:
Why is this added? This was removed in https://github.com/conda-forge/libnetcdf-feedstock/pull/56

Reply:
Missed that :-/
I'll fix it in the next bump.